src/Plugin/Drupal8/Audit/Database/DatabaseConnectionAnalysis.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Drutiny\Plugin\Drupal8\Audit;
+
+use Drutiny\Audit\AbstractAnalysis;
+use Drutiny\Sandbox\Sandbox;
+
+/**
+ * Provides all database connection information.
+ */
+class DatabaseConnectionAnalysis extends AbstractAnalysis {
+
+  public function gather(Sandbox $sandbox) {
+    $db_connections = $this->target->getService('drush')
+      ->runtime(function() {
+        return \Drupal\Core\Database\Database::getAllConnectionInfo();
```

Comment:
Same as other comments in this review, lets scrub the credentials we don't want to expose before returning them.

Reply:
Done.